news/2024/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area.md

```diff
@@ -0,0 +1,307 @@
+---
+layout: post
+title: "The Followpoint: xootynator, the CEO of Full Area"
```

Comment:
was told that letting the interviewees choose their own news title doesn't seem right

maybe
```suggestion
title: "The Followpoint: xootynator, the Full Area Appreciator"
```

though i wouldn't particularly mind
```suggestion
title: "The Followpoint: xootynator, the Full Area Stan"
```

Reply:
Figured I'd repeat here what I said on Discord. If there is no actual issue with the title excluding the fact it was suggested by xooty (actually not even her, but her moderator, she just told me) - I am against changing it, especially to something worse. I liked the title myself, as well as Utiba who I discussed it with, and felt like it's a better one to use that original, so I don't see a problem with keeping it.

Reply:
Revert to original IMO.

Reply:
@peppy sure, but may I ask why? Is the reason really just that it wasn't my idea?

Reply:
> was told that letting the interviewees choose their own news title doesn't seem right

Reply:
So it really is the only reason, understood